deployment/group_vars/wetc.yml

```diff
@@ -1,24 +1,39 @@
 ## General settings
 BRIDGE_MODE: "NATIVE_TO_ERC"
+HOME_NATIVE_NAME: "POA"
```

Comment:
Based on the previous comments. Configuration parameters could be taken from [the real UI](https://wetc.app/)

Reply:
Updated the values as suggested.